web/app/controllers/authenticated/all.ts

```diff
@@ -1,8 +1,4 @@
 import Controller from "@ember/controller";
```

Comment:
Do we need this file anymore?

Reply:
It's needed to pass the queryParams to the `/documents` route on redirect. I've added a note to the controller.